frontend/app/controllers/Joiner.scala

```diff
@@ -204,6 +204,8 @@ trait Joiner extends Controller with ActivityTracking with LazyLogging {
       case _ => Future.successful(None)
     }
 
+    val cookiesToDiscard = List(DiscardingCookie("GU_MEM"), DiscardingCookie("gu_paying_member", "/", Some(Config.guardianShortDomain)))
```

Comment:
I was thinking that if the set of cookies we discard when changing payment status is always the same we could define that entire list in Config.scala